src/game.ts

```diff
@@ -346,37 +330,20 @@ export class Game {
         this.players.forEach((player) => {
             player.elem.style.left = player.posX + "px";
             player.elem.style.top = player.posY + "px";
+            if (player.blastCounter > 0) {
```

Comment:
This logic makes more sense within the player update method since it's affecting the player